src/lib.rs

```diff
@@ -63,4 +64,6 @@ pub mod tokio;
 #[doc(inline)]
 pub use self::config::ReconnectOptions;
 #[doc(inline)]
+pub use self::strategies::ExpBackoffStrategy;
```

Comment:
I would probably just keep the strategies mod as public, not sure if we need to directly expose the struct here.